python/test/test_polars.py

```diff
@@ -1,8 +1,8 @@
 import pytest
 import opendp.prelude as dp
-import os
 import warnings
 import re
+import os
```

Comment:
It hasn't felt like a priority for me, but if we wanted cleaner diffs, we could use isort, perhaps as a precommit hook. I won't file an issue right now, but feel free.